src/ledger-db/initdb/1_create_transactions.sh

```diff
@@ -30,30 +31,36 @@ fi
 add_transaction() {
     DATE=$(date -u +"%Y-%m-%d %H:%M:%S.%3N%z" --date="@$(($6))")
     echo "adding default transaction: $1 -> $2"
-    psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" --dbname "$POSTGRES_DB" <<-EOSQL
+    psql -X -v ON_ERROR_STOP=1 -v fromacct="$1" -v toacct="$2" -v fromroute="$3" -v toroute="$4" -v amount="$5" --username "$POSTGRES_USER" --dbname "$POSTGRES_DB" <<-EOSQL
         INSERT INTO TRANSACTIONS (FROM_ACCT, TO_ACCT, FROM_ROUTE, TO_ROUTE, AMOUNT, TIMESTAMP)
-        VALUES ($1, $2, $3, $4, $5, '$DATE');
+        VALUES (:'fromacct', :'toacct', :'fromroute', :'toroute', :'amount', '$DATE');
 EOSQL
 }
 
-PAY_PREIODS=3
+
+PAY_PERIODS=3
 DAYS_BETWEEN_PAY=14
 SECONDS_IN_PAY_PERIOD=$(( 86400 * $DAYS_BETWEEN_PAY  ))
 echo $SECONDS_IN_PAY_PERIOD
 DEPOSIT_AMOUNT=250000
+PAYMENT_ACCOUNTS=("$DEFAULT_CONTACT_ACCOUNT_A" "$DEFAULT_CONTACT_ACCOUNT_B")
 
-START_TIMESTAMP=$(( $(date +%s) - $(( $(($PAY_PREIODS+1)) * $SECONDS_IN_PAY_PERIOD  ))  ))
-for i in $(seq 1 $PAY_PREIODS); do
+# create a UNIX timestamp in seconds since the Epoch
+START_TIMESTAMP=$(( $(date +%s) - $(( $(($PAY_PERIODS+1)) * $SECONDS_IN_PAY_PERIOD  ))  ))
+
+for i in $(seq 1 $PAY_PERIODS); do
     # create deposit transaction
-    add_transaction $DEFAULT_DEPOSIT_ACCOUNT $DEFAULT_USER_ACCOUNT $DEFAULT_DEPOSIT_ROUTING $LOCAL_ROUTING_NUM $DEPOSIT_AMOUNT $START_TIMESTAMP
+    add_transaction "$DEFAULT_DEPOSIT_ACCOUNT" "$DEFAULT_USER_ACCOUNT" "$DEFAULT_DEPOSIT_ROUTING" "$LOCAL_ROUTING_NUM" $DEPOSIT_AMOUNT $START_TIMESTAMP
+
     # create payments
     TRANSACTIONS_PER_PERIOD=$(shuf -i 3-11 -n1)
     for p in $(seq 1 $TRANSACTIONS_PER_PERIOD); do
         AMOUNT=$(shuf -i 100-25000 -n1)
-        ACCOUNT=$(shuf -i 1000000000-9999999999 -n1)
+        RECIPIENT_ACCOUNT=${PAYMENT_ACCOUNTS[$RANDOM % ${#PAYMENT_ACCOUNTS[@]}]}
```

Comment:
Aesthetically, it seems a little strange with all the payments going to only two accounts. Maybe we should consider adding more contacts, or add some random `None` payments in there?

I'm fine with how it is though too, looks great!

Reply:
Yep, I thought about that a bit.  I think I'll be able to address this when I update how we generate our default data via #131